tests/baselines/localhost_12345/admin/auth/user/#0.json

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
We need the @django/accessibility team to have a look at one of these outputs and confirm that what it is flagging up is valid, understandable, useful and actionable. 
If we want to use this tool, this report should be identifying real issues here we want to solve :+1:

Reply:
You can also run a webserver on the `results` folder and see the `*.html` files. They provide a good HTML report with intuitive UI which will help.